website/docs/add-secure-apps/applications/manage_apps.md

```diff
@@ -20,13 +20,25 @@ To add an application to authentik and have it display on users' **My applicatio
 
 2. Click **Create with Wizard**. (Alternatively, use our legacy process and click **Create**. The legacy process requires that the application and its authentication provider be configured separately.)
 
-3. In the **New application** wizard, define the application details, the provider type and configuration, and then click **Submit**.
+3. In the **New application** wizard, define the application details, the provider type, bindings for the application.
```

Comment:
Suggestion: "... and optional bindings for the application."  This mirrors the "... and optional UI settings" used down on line 25.